tests/bootstrap.php

```diff
@@ -88,7 +88,7 @@
 
 // Ensure default test connection is defined
 if (!getenv('db_dsn')) {
-    putenv('db_dsn=sqlite://' . TMP . 'debug_kit_test.sqlite');
+    putenv('db_dsn=sqlite:///' . TMP . 'debug_kit_test.sqlite');
```

Comment:
according to php.net only a single / would do, but the gist shows it really needs the extra /